doc/source/contribution/documentation/introduction.rst

```diff
@@ -23,15 +23,15 @@ Debian GNU/Linux, Ubuntu:
    % sudo pip install -r doc/requirements.txt
    % (cd doc && bundle install)
 
-CentOS, Fedora:
+AlmaLinux, Fedora:
 
 .. code-block:: console
 
    % sudo yum install -y python-pip gettext
```

Comment:
Ah, we should use `dnf` instead of `yum` on AlmaLinux and Fedora. We should fix it in a separated PR.

Reply:
Although I wonder whether it will works or not if we change the command from `dnf` to `yum`, I just know it works because we release the packages to AlmaLinux.

Reply:
AlmaLinux用Groongaパッケージをリリースしているからと言っていますか？

であればそれは関係ないです。`python-pip`も`gettext`もGroongaパッケージと関係ないからです。

`yum`はdeprecatedになって移行先が`dnf`です。互換性のために`yum`は`dnf`へのシンボリックリンクになっていますが、実体は`dnf`なので`yum`で動いていたら`dnf`でも動くはずです。

Reply:
> AlmaLinux用Groongaパッケージをリリースしているからと言っていますか？

はい。上記の理解で行っていました。

> 互換性のためにyumはdnfへのシンボリックリンクになっていますが、実体はdnfなのでyumで動いていたらdnfでも動くはず

知りませんでした。なるほどです！

Reply:
fix: https://github.com/groonga/groonga/pull/1725 `dnf` を利用する形に更新した！